05_syntax_lab/07.py

```diff
@@ -1,8 +1,27 @@
-""" Write a program that selects a random number
-and asks the user to guess it.
-
-After each guess print a hint "too large" or "too small" to the user.
-
-Bonus: To make things interesting, the program should cheat once in a white
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 1
+
+"""
+from random import randint
+random = randint(1, 100)
+fake = 0
+
+while True:
+    print "Enter number: "
+    num = int(raw_input())
+    fake += 1
+    if (num > random):
+        if (fake % 10) == 0:
+            print "Try to multiple by ?"
+        else:
+             print "Too big"
+    elif (num < random):
+        print "Too small"
+    else: break
+
+print "Bingo!"

```

Comment:
Keep in mind that it's now easy to guess when computer is cheating: Once every 10 rounds and only if the guess happens to be larger than number.

If you'll have some extra time after finishing the modules exercise, perhaps you can go back to this one and make it harder for a user to know if the game's cheating